src/main/java/lottogame/domain/lotto/Lottos.java

```diff
@@ -1,33 +1,37 @@
 package lottogame.domain.lotto;
 
-import lottogame.domain.dto.LottoResult;
+import lottogame.domain.statistic.Rank;
+import lottogame.domain.dto.LottoDto;
+import lottogame.domain.statistic.LottoResults;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Objects;
 import java.util.stream.Collectors;
 
 public class Lottos {
-    private final List<Lotto> lottos = new ArrayList<>();
+    private final List<Lotto> lottos;
 
     public Lottos(List<Lotto> values) {
-        lottos.addAll(new ArrayList<>(values));
+        List<Lotto> allLottos = new ArrayList<>();
+        allLottos.addAll(new ArrayList<>(values));
+        this.lottos = new ArrayList<>(allLottos);
```

Comment:
```suggestion
        this.lottos = new ArrayList<>(values);
```
요렇게만 해도 원하신 효과를 얻을 수 있을거에요~!